lib/rules/selector-max-universal/__tests__/index.js

```diff
@@ -228,6 +269,93 @@ testRule({
 	],
 });
 
+testRule({
+	ruleName,
+	config: [0, { ignoreAfterCombinators: ['+'] }],
+
+	accept: [
+		{
+			code: 'foo {}',
+		},
+		{
+			code: 'foo + * {}',
+		},
+	],
+
+	reject: [
+		{
+			code: '* {}',
+			message: messages.expected('*', 0),
+			line: 1,
+			column: 1,
+			endLine: 1,
+			endColumn: 2,
+		},
+	],
+});
+
+testRule({
+	ruleName,
+	config: [1, { ignoreAfterCombinators: ['>'] }],
+
+	accept: [
+		{
+			code: '*.foo {}',
+		},
+		{
+			code: '*.foo > * {}',
+		},
+	],
+
+	reject: [
+		{
+			code: '*.foo * {}',
+			message: messages.expected('*.foo *', 1),
+			line: 1,
+			column: 1,
+			endLine: 1,
+			endColumn: 8,
+		},
+	],
+});
+
+testRule({
+	ruleName,
+	config: [0, { ignoreAfterCombinators: ['~', '+', '>', 'h1', 'article'] }],
```

Comment:
[suggest] The `h1` and `article` cases may be unnecessary since they don't match the `ignoreAfterCombinators` name and may cause confusion. I think `'~', '+', '>'` is sufficient.

```suggestion
	config: [0, { ignoreAfterCombinators: ['~', '+', '>'] }],
```